### DISCUSSION TITLE
how to uninstall Onion OS

### DISCUSSION THREAD

**Discussion by marc0sfire** [OnionUI/Onion#1687] [Support]
I'm new to portable gaming devices, and since I didn't like the system that came with my RG35XX, I decided to install ONION OS as recommended by a YouTuber. I followed the whole process, and it worked for a while. However, now the games are freezing, making it impossible to play any game. So, I decided to remove ONION OS by formatting my SD card and installing a new system called MU OS. But now, my RG35XX won't turn on, neither with the original SD card nor with the one that has MU OS. I formatted the SD card again and reinstalled ONION OS, and the RG35XX turned back on. My question is: Is there a correct way to uninstall ONION OS in order to install another system?

**Comment by thomasklemenc** [comment]
Onion OS does not work on any devices but the Miyoo Mini / Miyoo Mini+
It does not work on RG35XX or any other device.
There is no "uninstallation" you can do besides deleting the Onion files on the SD card. You probably have to reinstall the original OS, but we cannot help you with that, sorry. 
You could try to get help on the muOS or the Retro Game Handhelds discord server.

**Reply by marc0sfire** [reply]
Thanks tGecko, i will try with the muOS community ✌️
